### PR TITLE
Zookeeper integration

### PR DESCRIPTION
**Pull request by enxebre**
This is pending on https://github.com/Capgemini/mesos-ui/pull/49 to be merged.
Provides a Zookeeper component for leader redirection.

### CODE REVIEW

**Comment by tarnfeld** [comment]
This is awesome :smile: :raised_hands:

File: src/components/ZookeeperRedirect/ZookeeperRedirect.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  }
+
+  // method mostly copy from https://github.com/apache/mesos/blob/master/src/webui/master/static/js/controllers.js
+  redirectToLeader() {
+    if (this.props.leader) {
+
+      // Redirect if we aren't the leader.
+      if (this.props.leader != this.props.pid) {
+
+        let leaderAlertDiv = document.querySelector('#not-leader-alert');
```
Comment (tayzlor) [inline src/components/ZookeeperRedirect/ZookeeperRedirect.js]: You should be able to attach a ref to the DOM node and use ```ReactDOM.findDOMNode``` here.

See http://facebook.github.io/react/docs/top-level-api.html#reactdom.finddomnode

File: src/components/ZookeeperRedirect/ZookeeperRedirect.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+        if (leaderAlertDiv != null) {
+          leaderAlertDiv.classList.remove('hide');
+          var redirectTime = this.props.redirectTime
```
Comment (tayzlor) [inline src/components/ZookeeperRedirect/ZookeeperRedirect.js]: use ```let``` instead of ```var```

Comment (enxebre) [inline src/components/ZookeeperRedirect/ZookeeperRedirect.js]: with Let wouldn't be available inside the countdown scope, we could probably just pass it as parameter

Comment (wallies) [inline src/components/ZookeeperRedirect/ZookeeperRedirect.js]: let have Block Scope, while variables declared with var have Function Scope

**Comment by enxebre** [comment]
This need bit more testing, don't merge yet.

**Comment by enxebre** [comment]
Added Reacjs friendly sauce and unit testing. I'll create a follow up for moving this in to a separate repo so we can just consuming it as a npm Package.

**Comment by tayzlor** [comment]
update changelog?

**Comment by enxebre** [comment]
I am a visionary, zookeeper is already in the changelog

**Pull request merged**